server/src/repositories/person.repository.ts

```diff
@@ -95,6 +95,7 @@ export class PersonRepository implements IPersonRepository {
       .innerJoin('face.asset', 'asset')
       .andWhere('asset.isArchived = false')
       .orderBy('person.isHidden', 'ASC')
+      .addOrderBy('person.isFavorite', 'DESC')
```

Comment:
Is it expected to have people ordered by isFavorite by default?

Reply:
I would argue yes, for me that was the only reason to add this favorite feature: to be able to display favorite people at the start of the list, and that is all it does for now.

For Users that don't use the feature, the sorting will still be the same, since every person will have it set to false.